sdcm/sct_events/events_device.py

```diff
@@ -171,7 +169,12 @@ def outbound_events(self,
                         LOGGER.debug("%s: add filter %s with uuid=%s", self, obj, obj.uuid)
                         filters[obj.uuid] = obj
 
-                if obj_filtered or isinstance(obj, SystemEvent):
+                if isinstance(obj, SystemEvent):
```

Comment:
I'm not following the reason for this change there was a logic issue ? Or what is exactly the reason for this change ?

Reply:
This one is part of the commit "fix(event_device): stop evaluating event filters against event filters"

Reply:
As you can see the purpose is to stop evaluating filters agaist filters.